src/Sarif.UnitTests/Visitors/SarifCurrentToVersionOneVisitorTests.cs

```diff
@@ -69,9 +69,7 @@ private void VerifyCurrentToVersionOneTransformationFromResource(string v2InputR
                 File.WriteAllText(expectedFilePath, v1ExpectedLogText);
                 File.WriteAllText(actualFilePath, v1ActualLogText);
 
-
-                string errorMessage = string.Format(@"V2 conversion from V1 produced unexpected diffs for test: '{0}'.", v2InputResourceName);
```

Comment:
>errorMessage [](start = 23, length = 12)

This variable was never used, so this message never appeared.

Reply:
ok. this code needs to be converted to actually use the helpers in the filediff base class.

---
In reply to: [247374990](https://github.com/Microsoft/sarif-sdk/pull/1211#discussion_r247374990) [](ancestors = 247374990)

Reply:
I'll do it now, while I'm in here.

---
In reply to: [247377138](https://github.com/Microsoft/sarif-sdk/pull/1211#discussion_r247377138) [](ancestors = 247377138,247374990)

Reply:
Done! This SarifCurrentToVersionOneVisitorTests.cs is no 65 lines long :-) Take a look at the changes to FileDiffingTests.cs to see how it works.

---
In reply to: [247582997](https://github.com/Microsoft/sarif-sdk/pull/1211#discussion_r247582997) [](ancestors = 247582997,247377138,247374990)